Torch.Server/Views/ConfigControl.xaml

```diff
@@ -89,19 +89,19 @@
                         <CheckBox IsChecked="{Binding PauseGameWhenEmpty}" Content="Pause When Empty" Margin="3" />
                     </StackPanel>
                     <StackPanel Grid.Column="1" Margin="3">
-                        <Label Content="Mods" />
+                        <!--<Label Content="Mods" />
```

Comment:
If you're removing something don't just comment it out.  We have a version control system to keep track of history.

Reply:
Alright. I just left it there, because I ensured it's still functional when uncommented. But I'll remove it.